docs/components/card/index.html

```diff
@@ -0,0 +1,142 @@
+    class="code-snippets code-snippets-md"
+    sandbox="allow-scripts allow-same-origin"
+  >
+  </iframe>
+
+  <h2 class="margin-vertical-3 margin-top-5">Card with Image & Badge</h2>
+  <p>These types of card can be used for product details.</p>
+  <div class="cards-group margin-vertical-4 border-radius-sm cards-group-2">
+    <div class="card card-vertical border-radius-sm card-image card-shadow">
+      <div class="card-header">
+        <img
+          src="https://picsum.photos/seed/picsum/150/150"
+          alt=""
+          class="border-radius-sm"
+        />
+      </div>
+      <div class="card-content">
+        <h3>Card Title</h3>
+        <p>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolorum
+          corporis architecto saepe corrupti sunt illum fuga aliquam error
+          eveniet iste, maiores quaerat? Molestiae, porro? Esse iure sint.
+        </p>
+      </div>
+      <div class="card-footer">
+        <button class="button button-link button-small">Read More >></button>
+      </div>
+    </div>
+    <div class="card card-vertical card-image border-radius-md card-shadow">
+      <div class="card-header">
+        <span class="badge badge-success-light badge-pill">SALE</span>
+        <img
+          src="/assets/images/Alex_maquine_shoulder.jpg"
+          alt=""
+          class="border-radius-sm"
+        />
+      </div>
+      <div class="card-content">
+        <small>Category</small>
+        <h3 class="margin-vertical-2">Shoulder Bag</h3>
+        <p>Shoulder Bag by Luxury British fashion house Alexander McQueen.</p>
+      </div>
+      <div class="card-footer">
+        <div class="card-price">
+          <span class="offer-price">Rs. 2999</span>
+          <span class="old-price">Rs. 4999</span>
+        </div>
+        <div class="buttons-group">
+          <button class="button button-danger-outline button-floating">
+            <i class="fa-regular fa-heart"></i>
+          </button>
+          <button class="button button-success button-floating">
+            <i class="fa-solid fa-cart-plus"></i>
+          </button>
+        </div>
+      </div>
+    </div>
+    <div class="card card-vertical card-image border-radius-md card-shadow">
```

Comment:
I think you should give a line space here. As this `div` is for a new type of card. This will increase the readability of the code.

Reply:
Sure!